kernel/aster-nix/src/net/socket/ip/stream/mod.rs

```diff
@@ -489,12 +498,13 @@ impl Socket for StreamSocket {
     fn recvfrom(&self, buf: &mut [u8], flags: SendRecvFlags) -> Result<(usize, SocketAddr)> {
         debug_assert!(flags.is_all_supported());
 
-        poll_ifaces();
-        if self.is_nonblocking() {
-            self.try_recvfrom(buf, flags)
+        let recv_bytes = if self.is_nonblocking() {
+            self.try_recvfrom(buf, flags)?
         } else {
-            self.wait_events(IoEvents::IN, || self.try_recvfrom(buf, flags))
-        }
+            self.wait_events(IoEvents::IN, || self.try_recvfrom(buf, flags))?
+        };
+
+        Ok(recv_bytes)
```

Comment:
Well, I don't understand why this PR removes `poll_ifaces()` and defines `recv_bytes`. Neither change makes sense in the context of this PR.

For the `poll_ifaces()`-related change, I think we can do it in https://github.com/asterinas/asterinas/pull/815. So this PR does not need to touch this method, right?

Reply:
The `poll_ifaces()`-related change was removed from this PR.